roles/gpu_operator_bundle_from_commit/files/operator_bundle_image_builder_script.yml

```diff
@@ -0,0 +1,51 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    app: gpu-operator-ci
+  name: operator-bundle-image-builder-script
+  namespace: gpu-operator-ci
+data:
+  run_operator_bundle_image_builder.sh: |
+    #! /bin/bash
+    set -ex
+    set -o nounset
```

Comment:
Sloppy rebasing work @kpouget ! You sneaked this line in a separate commit, I want to suggest adding it only to find it in the full PR diff

Reply:
ack, I'll try to reshape the commits if it's modified back and forth in multiple commits!